src/sentry/api/endpoints/organization_release_details.py

```diff
@@ -186,6 +189,14 @@ def put(self, request, organization, version):
                         datetime=release.date_released,
                     )
 
+            analytics.record(
+                "release.set_commits",
+                user_id=request.user.id if request.user and request.user.id else None,
+                organization_id=organization.id,
+                project_ids=[project.id for project in release.projects.all()],
+                user_agent=request.META.get("HTTP_USER_AGENT", ""),
+            )
+
```

Comment:
So, we want to see how many people are using the `--local` feature. Seems like this would record a hit even when the user is using a repo based integration. Am I reading this right?